apps/dashboard/src/app/redirect/redirect.component.ts

```diff
@@ -8,17 +8,25 @@ declare var gapi: any;
 declare var google: any;
 
 const AUTHORIZED_REDIRECTS = ['nuclia-desktop://', 'http://localhost:4200'];
+const AUTHORIZED_REDIRECTS_REGEX = [/^chrome\-extension\:\/\/[a-z]+\/options\/options\.html$/];
```

Comment:
this name seems a bit off to me, doesn't seem to be an authorized redirects regexp, seems to be a from chrome extension regexp 🤔

Reply:
ok, at the moment I keep the name. If at some point we come up with a better name it can be changed.